src/Util/src/FMI3/fmi3_enums.c

```diff
@@ -51,6 +51,14 @@ int fmi3_base_type_enu_is_float(fmi3_base_type_enu_t baseType) {
     return 0;
 }
 
+int fmi3_base_type_enu_is_enum(fmi3_base_type_enu_t baseType) {
+    switch (baseType) {
+        case fmi3_base_type_enum: return 1;
+        default: return 0;
+    }
+    return 0;
```

Comment:
Makes more sense to have an `if` here, or even better change it to return `bool` perhaps? Then `return baseType == fmi3_base_type_enum`